src/main/webapp/search-meal-style.css

```diff
@@ -0,0 +1,72 @@
+/* Copyright 2020 Google LLC
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+     https://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License. */
+
+body {
+  background-image: url("image/background-main.jpg");
+  background-repeat: no-repeat;
+  background-size: 100%;
+  background-position: right top;
+  background-attachment: fixed;
+  font-family: sans-serif;
+}
+
+input[type=text], select {
+  width: 100%;
+  padding: 12px 20px;
+  margin: 8px 0;
+  display: inline-block;
+  border: 1px solid;
+  border-radius: 4px;
+  box-sizing: border-box;
+}
+
+input[type=submit] {
+  width: 100%;
+  background-color: rgb(136, 235, 140);
+  color: white;
+  padding: 14px 20px;
+  margin: 8px 0;
+  border: none;
+  border-radius: 4px;
+}
+
+#search-block {
+  width: 300px;
+  position: relative;
+  top: 50%; 
+  margin: auto;
+  border-radius: 15px;
+  background-color: #ffffff;
+  padding: 20px;   
+}
+ul {
+  list-style-type: none;
+  padding: 0;
+  border: 1px solid rgb(199, 199, 199);
+}
+
+ul li {
```

Comment:
You can use the same CSS file for multiple pages. Also you can use multiple CSS files for a single file.

Seeing that `ul`, `ul li` styles and a couple others are identical to ones in `meal-page-style.css`, I'd like you to make a CSS file that holds all shared style definitions.
With that, each page can either use just the shared CSS (or multiple shared CSS files if you subdivide the content), or use the shared one plus one CSS file that holds a couple style statements that should apply only to that page.

Reply:
I must have done something wrong and one of my comments wasn't submitted.

In short, I agree with Bartek here.

It's very common to have a single CSS file for the whole webapp.
You'd normally define 3 kinds of css rules:
- the most generic rules, shared by the whole app - defined for [type selectors](https://developer.mozilla.org/en-US/docs/Web/CSS/Type_selectors), e.g. `body, li, ul, div`. You'd define stuff like fonts, font sizes, text decoration etc. on this layer
- more specific properties that only apply to a subset of a given type, or shared by some elements of variuous kinds - defined for [class selectors](https://developer.mozilla.org/en-US/docs/Web/CSS/Class_selectors). You can define generic classes, like `wide-button`, `green`, `with-fancy-border` that add some properties that are not common for all elements of a given kind. One classic example is for buttons: you'd often have two buttons, one with a positive action (e.g. Comment) and one with a negative one (e.g. Cancel). You'd define two classes, like 
```
.button-accept {
  color: green
}
.button-cancel {
 color: red
}
```
and apply those styles to each button individually by setting the `class=button-accept` parameter in HTML. You can actually see an example on Github. In the comment form I'm filling right now, the "Comment" button is different than the "Cancel" button, because it has an extra class defined: `btn-primary`.
- very specific properties that apply to only one element on a page are defined with [id selectors](https://developer.mozilla.org/en-US/docs/Web/CSS/ID_selectors). We've covered that above with the `body` element.